phpBB/phpbb/search/backend/base.php

```diff
@@ -24,18 +24,12 @@
 * optional base class for search plugins providing simple caching based on ACM
 * and functions to retrieve ignore_words and synonyms
 */
-class base
+abstract class base
 {
 	var $ignore_words = array();
```

Comment:
This variables look unused i think

Reply:
Yes, seems to be. I think it's only mentioned once in comments.